test/units/inventory/test_host.py

```diff
@@ -69,10 +69,10 @@ def test_get_groups(self):
 
     def test_equals_none(self):
         other = None
-        self.hostA == other
-        other == self.hostA
-        self.hostA != other
-        other != self.hostA
+        assert self.hostA == other
+        assert other == self.hostA
```

Comment:
I think this was supposed to be
```suggestion
        assert not (self.hostA == other)
        assert not (other == self.hostA)
```